.vscode/launch.json

```diff
@@ -64,5 +64,19 @@
             "internalConsoleOptions": "openOnSessionStart",
             "smartStep": true
         },
+        {
+            "name": "Debug gulpfile",
+            "type": "node",
+            "request": "launch",
+            "protocol": "inspector",
+
+            "cwd": "${workspaceFolder}",
+            "program": "${workspaceFolder}/node_modules/gulp/bin/gulp.js",
+            "args": [
+                "build",
+                "-LLLL"
```

Comment:
What's this arg?

Reply:
Removing (BTW: That makes gulp log in verbose mode :p )